src/ClientHandle.cpp

```diff
@@ -3354,10 +3375,10 @@ void cClientHandle::ProcessProtocolInOut(void)
 		cCSLock Lock(m_CSOutgoingData);
 		std::swap(OutgoingData, m_OutgoingData);
 	}
-	auto link = m_Link;
-	if ((link != nullptr) && !OutgoingData.empty())
+
+	if ((m_Link != nullptr) && !OutgoingData.empty())
```

Comment:
Why the unrelated change?

The original author was probably worried about `m_Link` being reset, so this could be introducing a race condition.